content/home/talks.md

```diff
@@ -25,7 +25,8 @@ exclude_selected = false
 
 **[CVPIA SDM](http://cvpia.scienceintegrationteam.com/meetings/)**     
 Conference call     
-September 11, 2019 (10 am - 12 pm)   
+September 25, 2019 (10 am - 12 pm)   
 **[Join Zoom](https://oregonstate.zoom.us/j/3571021423    )**
 Meeting ID: 357 102 1423  
-**[Agenda](http://cvpia.scienceintegrationteam.com/meetings/#agenda )**
\ No newline at end of file
+**Agenda**    
```

Comment:
don't change the agenda link. if someone clicks on it, they will be redirected to the meetings page which will say coming soon. this way you won't have to keep editing it back and forth